ggml-rs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "ggml-rs"
```

Comment:
I would keep this as `ggml` - I personally dislike the `-rs` suffix because it remaps to `_rs` in the crate.

Reply:
All set ✅